doc/releases/quincy.rst

```diff
@@ -5,6 +5,1458 @@ Quincy
 Quincy is the 17th stable release of Ceph. It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+
+
+v17.2.8 Quincy
+==============
+
+This is the eighth backport release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
+* RADOS: `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated
+  due to being prone to false negative results.  It's safer replacement is
+  `pool_is_in_selfmanaged_snaps_mode`.
+* RBD: When diffing against the beginning of time (`fromsnapname == NULL`) in
+  fast-diff mode (`whole_object == true` with `fast-diff` image feature enabled
+  and valid), diff-iterate is now guaranteed to execute locally if exclusive
```

Comment:
locally on the cluster, or on the client?

Reply:
don't know what this is :(

Reply:
On the client -- this says that if certain conditions are met, the client wouldn't need to send any OSD requests to service a diff-iterate request.

Reply:
@idryomov what do you want to change in the paragraph above?
Or do you at all?

Reply:
I'm fine with it as is -- it's already out there in pacific, reef and squid release notes.